specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -240,7 +240,8 @@ export function IntegratedRecordSelector({
                 preHeaderButtons={collapsibleButton}
                 sortField={sortField}
                 viewName={viewName}
-                onAdd={(): void => {
+                onAdd={(resources): void => {
+                  if (!isInteraction) collection.add(resources);
                   if (typeof handleAdd === 'function') handleAdd();
                 }}
```

Comment:
you shouldn't be calling collecion.add here as it's now part of handleAdd

```suggestion
                onAdd={handleAdd}
```

Reply:
the handleAdd() in the FormTableCollection props is not called from line 122 (the onAdd prop of RecordSelectorFromCollection) but from line 132 in FormTable. Hence the need to keep this here. 
https://github.com/specify/specify7/blob/9044ffc1e29eea13e1d18ccf20cad10ef5290978/specifyweb/frontend/js_src/lib/components/FormCells/FormTable.tsx#L132

Reply:
I tested this solution and it works for all cases it seems. (interactions, subviews, formTable, etc)